ironfish/src/blockchain/blockchain.ts

```diff
@@ -268,6 +273,8 @@ export class Blockchain<
 
       return { isAdded: true, reason: null }
     })
+
+    return { ...result, score: result.reason ? BAN_SCORE.MAX : BAN_SCORE.NO }
```

Comment:
This won't be hard coded later, it'll be returned from all the validation functions.